Neos.Utility.Lock/Classes/FlockLockStrategy.php

```diff
@@ -146,7 +146,7 @@ protected function applyFlock($exclusiveLock)
      *
      * @return boolean TRUE on success, FALSE otherwise
      */
-    public function release()
+    public function release(): bool
```

Comment:
Apparently it's safe to typehint return types in implementations even though they're not present in the interface.